dev/installer-diff/cmd/filter.go

```diff
@@ -133,7 +153,7 @@ func filterGenericStuff(obj *unstructured.Unstructured) (err error) {
 	// (needs to be below the kind-specific stuff to override converter artifacts!)
 	obj.SetAnnotations(emptyMap)
 	obj.SetNamespace("")
-	obj.SetCreationTimestamp(v1.Time{})	// results in "null". How to set to "nil" w/o writing by hand?
+	obj.SetCreationTimestamp(v1.Time{}) // results in "null". How to set to "nil" w/o writing by hand?
```

Comment:
Looks like it's not possible. It ends up with value of `null` because you're giving time of 0 (default value) but the API looks very determined to ensure this isn't set to an invalid value. I couldn't find a good way to do this.

Reply:
> Looks like it's not possible

Jep. Setting to a common value works well for us. :+1: